shell/app/common/components/authorize-member-modal/index.tsx

```diff
@@ -120,6 +120,7 @@ const AuthorizeMemberModal = ({ type, member, closeModal }: IProps) => {
     <Modal
       title={i18n.t('authorize user {user}', { user: member ? member.nick || member.name : '' })}
       visible={!!member}
+      centered
```

Comment:
I dont recommend to modify all use case. This may cause one problem that the next guy who use modal would forget this. So my suggestion is
1. define a custom modal in common which centered is true by default
2. add eslint check to prevent import modal from antd

```
"no-restricted-imports": ["error", {
      "paths": [{
        "name": "antd",
        "importNames": ["Modal"],
        "message": "Please use Modal from common instead."
      }]
    }]
```

Reply:
Add default props to Modal before all init logic:
```
Modal.defaultProps.centered = true;
```